### PR TITLE
Dev-5166 Loan Table Tab Tooltips

### PR DESCRIPTION
**Pull request by jonhill13**
**High level description:**

Adds tooltips for the loan table tabs and updates the column text.

**Technical details:**

> • column text updates for Advance Search, Keyword Search, Federal Accounts and Award.

> • tooltips for table tabs for Advance Search, Keyword Search, Federal Accounts and Award.

**JIRA Ticket:**
[DEV-5166](https://federal-spending-transparency.atlassian.net/browse/DEV-5166)

**Mockup:**
N/A

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR
- [x] Verified cross-browser compatibility
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension) Compared Dev to local for advanced search both resulting in 48 errors.
- [N/A] Added Unit Tests for methods in Container Components, reducers, and helper functions `if applicable`
- [x] All componentWillReceiveProps, componentWillMount, and componentWillUpdate in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) per [JIRA item 3339](https://federal-spending-transparency.atlassian.net/browse/DEV-3339)
- [N/A] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [N/A] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [N/A] Design review complete `if applicable`
- [N/A] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

File: src/_scss/pages/search/results/table/_tableTypes.scss [hunk 1]
```diff
@@ -54,6 +54,17 @@
                     background-color: $color-gold;    
                 }
             }
+              .tooltip-wrapper {
+                .tooltip__text {
+                    padding: 2rem 2rem 0 2rem;
```
Comment (MaxwellKendall) [inline src/_scss/pages/search/results/table/_tableTypes.scss:59]: `rem()`?!?

File: src/_scss/pages/search/results/table/_tableTypes.scss [hunk 2]
```diff
@@ -54,6 +54,17 @@
...
+                    strong {
+                      padding-bottom: 1rem;
+                    }
+                }
+                .tooltip__text:last-child {
+                  padding: 2rem;
```
Comment (MaxwellKendall) [inline src/_scss/pages/search/results/table/_tableTypes.scss:65]: ditto!

File: src/js/components/award/shared/InfoTooltipContent.jsx [hunk 3]
```diff
@@ -224,6 +224,34 @@ export const transactionHistoryInfoFinancialAssistance = (
...
+            government, taking into account the interest rate and the modeled risk of the
+            recipient failing to pay back the loan in part or full; subsidy cost can be
+            positive (indicating that the government is likely to lose money on the loan) or
+            negative (indicating that the government is likely to make money on the
+            loan). Subsidy cost should never be larger in absolute value terms than
+            the face value itself. Administrative costs of running the loan or loan
+            guarantee program itself are excluded from subsidy cost calculations. Note
+            that a loan&#39;s face value is not considered Federal spending, since it
+            does not in itself represent a long-term cost to the government.
+            </p>
```
Comment (MaxwellKendall) [inline src/js/components/award/shared/InfoTooltipContent.jsx:254]: Wow.

File: src/_scss/pages/search/results/table/_tableTypes.scss (hunk 1 above)
Comment (jonhill13) [reply src/_scss/pages/search/results/table/_tableTypes.scss:59]: i thought we were moving away from that because it is part of bourbon

**Pull request merged**